snowpack/src/build/build-pipeline.ts

```diff
@@ -181,7 +181,7 @@ async function runPipelineTransformStep(
           isDev,
           isPackage,
           fileExt: destExt,
```

Comment:
It’s important that `fileExt` not change! That would basically break all transform plugins